src/map/battle.cpp

```diff
@@ -1512,7 +1512,11 @@ int64 battle_calc_damage(struct block_list *src,struct block_list *bl,struct Dam
 			status_change_spread(bl, src, 1); // Deadly infect attacked side
 
 	} //End of target SC_ check
-
+	
+	if (status_get_race2(bl) == RC2_GREENAURA) {
+		if (status_get_race2(src) != RC2_GREENAURA) // [Sona]: This is for MvP vs MvP event purposes. (damage doesn't get reduced when attacking eachother)
```

Comment:
Just curious, could this case even happen where a monster attacks another? I can't think of anything on kRO that does this.

Reply:
I've seen some cases where bosses battle eachother if hit by a sanctuary of another one